src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerNoticeOfChange.java

```diff
@@ -102,6 +104,36 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
                 .done();
     }
 
+    public AboutToStartOrSubmitResponse<CaseData, State> midEvent(
+        final CaseDetails<CaseData, State> details,
+        final CaseDetails<CaseData, State> detailsBefore
+    ) {
+        CaseData data = details.getData();
+
+        if (NO.equals(data.getNoticeOfChange().getAreTheyRepresented())
```

Comment:
nit picking but this could be a method on NoticeOfChange to make it clearer what you are doing